code-challenges.test.js

```diff
@@ -14,16 +14,79 @@
+// let numbersArray2 = [24, 27, 30, 33, 36]
+
+// describe ("divisibleByThree",() => {
+//     it("takes in two arrays filters through them to determine if numbers are divisible by three", 
+//     () => {
+//         let numbersArray1 = [6, 7, 8, 9, 10]
+//         let numbersArray2 = [24, 27, 30, 33, 36]
+//     expect(divisibleByThree(numbersArray1,numbersArray2)).toEqual(["15 is divisible by three"])
+//     expect(divisibleByThree(numbersArray1,numbersArray2)).toEqual(["0 is divisible by three"])
+//     expect(divisibleByThree(numbersArray1,numbersArray2)).toEqual(["-7 is not divisible by three"])
+//     })
+// })
+
+// ---->  FAIL  ./code-challenges.test.js
+//   ● Test suite failed to run
+// Test Suites: 1 failed, 1 total
+// Tests:       0 total
+// Snapshots:   0 total
+// Time:        0.209 s
+// Ran all test suites.
+// error Command failed with exit code 1.
+
 // b) Create the function that makes the test pass.
+// psuedocode: creat function that takes in numbersArray1 and numbersArray2 and test them to see if they are divisable by 3. Return "15 is divisible by three", "0 is divisible by three", "-7 is not divisible by three".
+
+// const divisibleByThree = (numbersArray1,numbersArray2) => {
+//     {
+//     return ("15 is divisible by three") } => {
+//     return ("0 is divisible by three") } =>  {
+//     return ("-7 is divisible by three") }
+//         let divisibleByThree = numbersArray1.filter(number 3 %=== 0),(numbersArray2.filter(number 3 %=== 0))
+// }
+
+// ----> FAIL       52 | const divisibleByThree = (numbersArray1,numbersArray2) => {
+    //   53 |     {
+    //     > 54 |     return ("15 is divisible by three") } => {
+    //          |                                           ^
+    //       55 |     return ("0 is divisible by three") } => {
+    //       56 |     return ("-7 is divisible by three") }
+    //       57 | }
+    
 
 // --------------------1) Create a function that takes a object as an argument and decides if the number inside it is evenly divisible by three or not.
 
+
 // a) Create a test with expect statements for each of the variables provided.
+// psuedocode: create test describing a conductor as "divisibleByThree" and using 3 expect statements set .toEqual the output array for each of the variables provided. expect 
+// input: 2 arrays 
+// output: string that says given number "is/is not divisible by three"
+
+
+describe ("divisibleByThree",() => {
+        it("takes in two arrays filters through them to determine if numbers are divisible by three", 
+        () => {
+            let numbersArray1 = [6, 7, 8, 9, 10]
+            let numbersArray2 = [24, 27, 30, 33, 36]
+        expect(divisibleByThree(numbersArray1,numbersArray2)).toEqual(["15 is divisible by three"])
+        expect(divisibleByThree(numbersArray1,numbersArray2)).toEqual(["0 is divisible by three"])
+        expect(divisibleByThree(numbersArray1,numbersArray2)).toEqual(["-7 is not divisible by three"])
+        })
+    })
```

Comment:
Quick question, how much does it matter that i used let statements on lines 81 and 82 instead of const? would the difference be that my numbersArray1 and numbersArray2 would only be those numbers within that block and is only able to be read in my code block and if i used cons, it would be that way and be read anywhere throughout the rest of my code?  just trying to figure out if the best practice would be to use a const.